back/CinemaReservation.Web/Controllers/AccountController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+using CinemaReservation.PresentationLayer.Models;
+using CinemaReservation.BusinessLayer.Models;
+using CinemaReservation.BusinessLayer.Contracts;
+
+namespace CinemaReservation.PresentationLayer.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AccountController : Controller
+    {
+        private readonly IAccountService _accountService;
+        public AccountController(IAccountService accountService)
+        {
+            _accountService = accountService;
+        }
+
+
+        [HttpPost("register")]
+        public async Task<IActionResult> PostUserAccount(RegistrationRequest registrationRequest)
+        {
+            if (registrationRequest == null)
+            {
+                return StatusCode(400);
```

Comment:
BadRequest method of controller exists. Also HttpStatusCode enumeration exists. Is it possible to have null model here?